src/site/_collections/authors.js

```diff
@@ -81,23 +58,38 @@ module.exports = (collections) => {
 
     // Limit posts for percy
     if (process.env.PERCY) {
-      author.elements = author.elements.slice(-6);
+      elements = elements.slice(-6);
+    }
+
+    // Set created on date and updated date
+    if (elements.length > 0) {
+      date = elements.slice(-1).pop().data.date;
+      const tempUpdated = elements.slice(0, 1).pop().data.date;
+      if (date !== tempUpdated) {
+        updated = tempUpdated;
+      }
     }
```

Comment:
I see that we had this before but it's kind of confusing. What is it actually doing?

Reply:
Not on the author pages (though we could), but it is used by for algolia and eventually firebase indexing for push notifications.